src/main/java/uk/gov/companieshouse/exemptions/delta/delete/DeleteClient.java

```diff
@@ -36,19 +35,9 @@ void delete(DeleteRequest request) {
                     .deleteCompanyExemptionsResource(request.getPath())
                     .execute();
         } catch (ApiErrorResponseException e) {
-            if (e.getStatusCode() / 100 == 5) {
-                logger.error(String.format("Server error returned with status code: [%s] when deleting delta", e.getStatusCode()));
-                throw new RetryableException("Server error returned when deleting delta", e);
-            } else {
-                logger.error(String.format("Delete client error returned with status code: [%s] when deleting delta", e.getStatusCode()));
-                throw new NonRetryableException("DeleteClient error returned when deleting delta", e);
-            }
-        } catch (IllegalArgumentException e) {
-            logger.error("Illegal argument exception caught when handling API response");
-            throw new RetryableException("Server error returned when deleting delta", e);
+            handler.handle(e);
```

Comment:
These two exception catch blocks can we unified using `||`

Reply:
The ResponseHandler class handles these two exceptions separately.
To do this would require modification to the handle method (add conditional check on what type of exception it is).

Not sure if its worth that amount of effort to get rid of one line.